Gemfile.lock

```diff
@@ -440,4 +440,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   1.17.3
+   2.0.2
```

Comment:
can you revert this part. Bundler 2.0 causes a world of hurt our in builds

Reply:
reverted sorry